erpnext/docs/user/manual/en/accounts/journal-entry.md

```diff
@@ -68,4 +68,38 @@ of each year reducing its value by a certain percentage.
 > Note: There may be regulations in your country that define by how much
 amount you can depreciate a class of Assets.
 
+#### Credit Note
+
+Credit note is a note which is given to the customer when we get his returned 
```

Comment:
Try and use gender neutral language, also documentation is best written in 3rd person.

`"Credit Note" is made for a Customer against a Sales Invoice when the company needs to refund or adjust a payment. When a Credit Note is made, the seller can either make a payment to the customer or adjust the amount in another invoice.`